commit f81fd07bdf8cf9f87c603754e3e5b89ed5445bf8
Author: Andrew Price <anprice@redhat.com>
Date:   Thu Oct 17 13:12:31 2019 +0100

    fsck.gfs2(8): Manpage updates
    
    - Improve style consistency with the other manpages
    - Remove an unnecessary paragraph that gives a misleading impression of
      gfs2's device atomicity requirements (rhbz#1693000)
    - Add "See Also" section
    - "fsck.gfs" -> "fsck.gfs2"
    - Various other language tweaks
    
    Signed-off-by: Andrew Price <anprice@redhat.com>

diff --git a/gfs2/man/fsck.gfs2.8 b/gfs2/man/fsck.gfs2.8
index b2b326fb..9e9f9250 100644
--- a/gfs2/man/fsck.gfs2.8
+++ b/gfs2/man/fsck.gfs2.8
@@ -1,11 +1,11 @@
 .TH fsck.gfs2 8
 
 .SH NAME
-fsck.gfs2 - Offline GFS and GFS2 file system checker
+fsck.gfs2 - offline GFS and GFS2 file system checker
 
 .SH SYNOPSIS
 .B fsck.gfs2
-[\fIOPTION\fR]... \fIDEVICE\fR
+[\fIoptions\fR] \fIdevice\fR
 
 .SH WARNING
 All computers \fImust\fP have the filesystem unmounted before running
@@ -13,30 +13,22 @@ fsck.gfs2.  Failure to unmount from all nodes in a cluster will likely result
 in filesystem corruption.
 
 .SH DESCRIPTION
-fsck.gfs2 will check that the GFS or GFS2 file system on a device is structurally valid.
-It should not be run on a mounted file system.  If file system corruption is
-detected, it will attempt to repair the file system.  There is a limit to what
-fsck.gfs2 can do.  If important file system structures are destroyed, such that
-the checker cannot determine what the repairs should be, reparations could
-fail.
+fsck.gfs2 will check that the GFS or GFS2 file system on a device is
+structurally valid.  It should not be run on a mounted file system.  If file
+system corruption is detected, it will attempt to repair the file system.
+There is a limit to what fsck.gfs2 can do.  If important file system structures
+are destroyed, such that the checker cannot determine what the repairs should
+be, reparations could fail.
 
-GFS2 is a journaled file system, and as such should be able to repair damage to
-the file system on its own.  However, faulty hardware has the ability to write
-incomplete blocks to a file system thereby causing corruption that GFS2 cannot
-fix.  The first step to ensuring a healthy file system is the selection of
-reliable hardware (i.e. storage systems that will write complete blocks - even
-in the event of power failure).
-
-Note: Most file system checkers will not check the file system if it is
-"clean" (i.e. unmounted since the last use).  The fsck.gfs program behaves
-differently because the storage may be shared among several nodes in a
-cluster, and therefore problems may have been introduced on a different
-computer.  Therefore, fsck.gfs2 will always check the file system unless
-the -p (preen) option is used, in which case it follows special rules
+Other file system checkers will not check the file system if it is "clean"
+(i.e. unmounted since the last use).  With gfs2, storage may be shared among
+several nodes in a cluster, and therefore problems may have been introduced on
+a different computer.  Therefore, fsck.gfs2 will always check the file system
+unless the -p (preen) option is used, in which case it follows special rules
 (see below).
 
-fsck.gfs2 will log to the system log on start and exit to aid debugging and
-administration.
+fsck.gfs2 will log a message to the system log on start and exit to aid
+debugging and administration.
 .SH OPTIONS
 .TP
 \fB-a\fP
@@ -86,3 +78,8 @@ Yes to all questions. By specifying this option, fsck.gfs2 will not prompt befor
 changes.
 
 This option may not be used with the \fB-n\fP or \fB-p\fP/\fB-a\fP options.
+
+.SH SEE ALSO
+.BR gfs2 (5),
+.BR gfs2_jadd (8),
+.BR gfs2_grow (8)
